weeklymission2/src/Components/Header.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import logo from '../images/logo.svg';
+
+export function Header({ profile, folder }) {
+  return (
+    <>
+      <div className="headerArea">
+        <nav>
+          <div className="navBar">
+            <a href="index.html"
+              ><img src={logo} alt="logo" className="logo"
+            /></a>
+            {profile ?
+              (<div className="profile"><img src={profile.image} alt="userProfileImg" className="profileImg"></img>
+              <span className="userEmail">{profile.email}</span></div>) :
+              (<a href="../Pages/signIn.html" className="btn btnS">로그인</a>)
+            }
+          </div>
+        </nav>
+        <header>
+          <div className="folderProfile">
+            <img src={folder.profileImage} alt="폴더소유자프로필사진" className="folderImg"/>
```

Comment:
폴터 프로필이미지를 가져오기전까지는 다 null이죠?
디폴트 프로퍼티를 줘야합니다. 
```suggestion
            <img src={folder.profileImage || ""} alt="폴더소유자프로필사진" className="folderImg"/>
```

Reply:
이미지 src에 대해서 빈 문자열을 주는게 실무에서는 좋은 방법은 아닙니다.
실무에서는 src가 없을때에 대한 이미지가 준비되어 있기 때문에 디폴트 src는 사용자의 이미지가 없을때 보여주는 이미지에 대한 주소입니다.